app/service/data_svc.py

```diff
@@ -172,11 +172,15 @@ def __init__(self):
 
     async def _load(self):
         try:
+            # Loads abilities from each plugin first
             for plug in [p for p in await self.locate('plugins') if p.data_dir]:
```

Comment:
Javon, check it out, I can comment in the PR now. Anyway, after talking with William, I am slowly getting an understanding of what they want when it comes to code. I am in no way an authority on it, but I wanted to run some suggestions and questions by you.

At this point, is there a way to turn this into a function? Specifically, for plug in [p for p in await self.locate('plugins') if p.data_dir]:

That way, instead of calling it twice, just create it once and call the appropriate things for each event, namely the loading of abilities then loading everything else?